localization/german/wc_tooltip_l_german.yml

```diff
@@ -33,4 +33,14 @@
  is_isolated_in_pandaria_tooltip_first_not:0 "Die Nebel Pandarias erlauben Interaktionen zwischen [CHARACTER.GetShortUIName|U] und [TARGET_CHARACTER.GetShortUIName|U]"
  
  is_isolated_in_uldum_tooltip_first:0 "Der Schleier von Uldum erlaubt #bold keine#! Interaktionen zwischen [CHARACTER.GetShortUIName|U] und [TARGET_CHARACTER.GetShortUIName|U]"
- is_isolated_in_uldum_tooltip_first_not:0 "Der Schleier von Uldum erlaubt Interaktionen zwischen [CHARACTER.GetShortUIName|U] und [TARGET_CHARACTER.GetShortUIName|U]"
\ No newline at end of file
+ is_isolated_in_uldum_tooltip_first_not:0 "Der Schleier von Uldum erlaubt Interaktionen zwischen [CHARACTER.GetShortUIName|U] und [TARGET_CHARACTER.GetShortUIName|U]"
+  
+ shatter_realm_tooltip_first:0 "Your realm falls apart freeing [vassals|E] and giving lands to [claimants|E]"
+ shatter_realm_tooltip_first_past:0 "Your realm falled apart freeing [vassals|E] and giving lands to [claimants|E]"
+ shatter_realm_tooltip_global:0 "[CHARACTER.GetFirstNamePossessive|U] realm falls apart freeing [vassals|E] and giving lands to [claimants|E]"
+ shatter_realm_tooltip_global_past:0 "[CHARACTER.GetFirstNamePossessive|U] realm falled apart freeing [vassals|E] and giving lands to [claimants|E]"
+
+ leave_union_tooltip_global:0 "[CHARACTER.GetShortUIName|U] leaves the current [union|E]"
+ leave_union_tooltip_global_past:0 "[CHARACTER.GetShortUIName|U] left the current [union|E]"
```

Comment:
Looks like it's from another PR. You should clear it.

Reply:
@arithon

Reply:
Fixed in 972ddd2

Reply:
I intentionally added them so the file would be complete...